readme.md

```diff
@@ -112,19 +119,24 @@ $ npm -v
 * **Question:** What gets logged? Why?
 
 * **Your Answer:**
+my name gets logged because the module.exports = my name
 
 * **Question:** What is `module.exports` and what is its _type_ in JavaScript? What is `require` and what is its _type_ in JavaScript?
 
 * **Your Answer:**
-
+module.exports makes code available to use in other files / classes by referencing it there through the require key word
 ---
 
 - [ ] We can only export one thing from files when using Node. With that said, export both your name and your birthday from the `profile.js` file.
 
 * **Question:** What are some ways you can solve this problem?
 
 * **Your Answer:**
-
+export a json element such as
+module.exports = {
+    name : "Paul",
+    birthday : "01/01/0001"
+}
```

Comment:
This isn't JSON, this is JavaScript. JSON is JavaScript Object Notation -- it's essentially a string. But, correct, you can package up multiple values with an object.